rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -101,7 +100,8 @@ object RemainderNormalizeMatcher {
 
 object CollectionNormalizeMatcher {
   def normalizeMatch[M[_]](c: Collection, input: CollectVisitInputs)(
-      implicit sync: Sync[M]
+      implicit sync: Sync[M],
+      deployerPk: Option[PublicKey]
```

Comment:
well, shit... I guess let's have the normalizer pass the deployerPk via implicits. Please leave a comment on the normalizer's entry point saying:

```//FIXME before adding any more implicits, or fields to the `*VisitInputs` classes, make the normalizer use ApplicativeAsk / MonadState instead```

Reply:
Done